posthog/models/user.py

```diff
@@ -147,24 +172,7 @@ def organization(self) -> Optional[Organization]:
     @property
     def team(self) -> Optional[Team]:
         if self.current_team is None and self.organization is not None:
-            if (
-                AvailableFeature.PROJECT_BASED_PERMISSIONING not in self.organization.available_features
-                or self.organization.memberships.get(user=self).level >= OrganizationMembership.Level.ADMIN
-            ):
-                # If project access control is NOT applicable, simply prefer open projects just in case
-                self.current_team = self.organization.teams.order_by("access_control", "id").first()
-            else:
-                # If project access control IS applicable, make sure the user is assigned a project they have access to
-                from ee.models import ExplicitTeamMembership
-
-                available_private_project_ids = ExplicitTeamMembership.objects.filter(
-                    parent_membership__user=self
-                ).values_list("team_id", flat=True)
-                self.current_team = (
-                    self.organization.teams.filter(Q(access_control=False) | Q(pk__in=available_private_project_ids))
-                    .order_by("access_control", "id")
-                    .first()
-                )
+            self.current_team = self.teams.first()
```

Comment:
This doesn't use `self.organization` anymore, which could result in `current_team` being set to a team belonging to an org different than `current_organization`.